install_config/storage_examples/privileged_pod_storage.adoc

```diff
@@ -0,0 +1,202 @@
+= Mount Volumes on Privileged Pods
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap:
+
+== Overview
+
+The following provides a method for mounting persistent volumes to pods attached
+to the *privileged scc*.  For the purpose of this guide, GlusterFS will be the
+sample use-case though this method can be adapted to any
+link:../install_config/persistent_storage/index.html[supported storage plugin].
+
+=== Prerequisites
+* A gluster volume exists and *glusterfs-fuse* is installed on all hosts
+* Definitions written for GlusterFS link:../persistent_storage/persistent_storage_glusterfs.html#creating-gluster-endpoints[endpoints and service],
+link:../persistent_storage/persistent_storage_glusterfs.html#gfs-creating-persistent-volume[persistent volume]
+and
+link:../architecture/additional_concepts/storage.html#persistent-volume-claims[persistent volume claim]
+* A user with the
+link:../admin_guide/manage_authorization_policy.html#managing-role-bindings[cluster-admin] role binding.
+For this guide, that user is called `admin`
+
+[[create-basic-user]]
+== Create a basic-user
+
+Create the basic-user by logging in.
+
+[source]
+----
+$ oc login -u tom -p tom
+----
+
+NOTE: Here, `tom` is an arbitrary username and password
+
+Next, create the project.
+
+[source]
+----
+$ oc new-project <project_name>
+----
+
+Basic-users are bound to the project-admin role at project creation so there is
+ no need to manually bind them.
+
+[[edit-privileged-scc]]
+== Edit the privileged scc
+
+TIP: For pods not run as privileged, skip this section.
+
+The basic-user must be added to the privileged scc (or to a group given access
+  to the scc) before they can run privileged pods.
+
+.*As admin*
```

Comment:
I would just make this a sentence instead of a title/bold. Also you can use `oadm policy add-scc-to-user` to do this in one step. e.g.:

```
Switch back to the *admin* user, then add the *privileged* SCC to the *tom* user:

----
$ oadm policy add-scc-to-user privileged tom
----
```